src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/tests/test_sql_commands.py

```diff
@@ -341,6 +341,33 @@ def body(self):
                      JMESPathCheck('elasticPoolName', self.pool_name),
                      JMESPathCheck('status', 'Online')])
 
+        # test sql elastic-pools db sub-group commands
+        self.cmd('sql elastic-pools db list -g {} --server-name {} --elastic-pool-name {}'
+                 .format(rg, self.sql_server_name, self.pool_name),
+                 checks=[
+                     JMESPathCheck('length(@)', 1),
+                     JMESPathCheck('[0].resourceGroup', rg),
+                     JMESPathCheck('[0].name', self.database_name),
+                     JMESPathCheck('[0].elasticPoolName', self.pool_name),
+                     JMESPathCheck('[0].status', 'Online')])
+
+        self.cmd('sql elastic-pools db show -g {} --server-name {} --elastic-pool-name {} '
+                 '--database-name {}'
```

Comment:
Generally speaking, I prefer whatever means to reduce users' keystrokes. But here `name` would be a little bit confusing. Does it refers to the database or does it refer to the elastic pool? In addition, I believe to change the name requires additional work to redefine the parameter. I don't think the work justify the value.